src/fragments/lib/storage/js/download.mdx

```diff
@@ -137,6 +137,42 @@ You can use `expires` option to limit the availability of your URLs. This config
 await Storage.get('filename.txt', { expires: 60 });
 ```
 
+### Example
+
+Below is an example of a React component with a Download link to a file:
```

Comment:
In general, do we want to extend our js examples with framework specific versions? This is inconsistent with our other documentation and we should review this to ensure it aligns with our docs strategy.

This wraps the “[File download option](https://docs.amplify.aws/lib/storage/download/q/platform/js/#file-download-option)” example in React, extending it to show the user a download link. Should we recommend developers offer file download using the `download` flag?

If we're looking to provide an example of how to download a file using storage, wouldn't an example using an `<a href...>`  like the one detailed under "[Get](https://docs.amplify.aws/lib/storage/download/q/platform/js/#get)" be better for our customers so our their users are allowing the browser to handle the file download instead of pulling the file contents into memory before triggering the download?